configs/application.conf

```diff
@@ -0,0 +1,98 @@
+###################################
+# Common configuration directives #
+###################################
+
+common-dc-port = 6020
+common-pg-port = 5432
+common-host = "local.dev.socrata.net"
+common-backup-host = ${common-host}
+common-soql-server-pg-host = ${common-host}
+common-pg-host = ${common-host}
+common-zk-ensemble = ["local.dev.socrata.net:2181"]
```

Comment:
I would remove all these `common-` things and make a cleaner config. Actually, I already have a branch out creating a configs/application.conf file for DC